lib/convert.js

```diff
@@ -66,17 +66,20 @@ async function convert({ paths, lossless, avif, webp, force }) {
   const totalSize = { before: 0, after: 0 };
   const tasksResult = await Promise.all(tasks);
 
-  tasksResult.forEach(({ fileBuffer, filePath, outputFormat } = {}) => {
-    const fileSize = {
-      before: fs.statSync(filePath).size,
-      after: fileBuffer.length,
-    };
-
-    totalSize.before += fileSize.before;
-    totalSize.after += fileSize.after;
+  tasksResult
+    .filter(Boolean)
```

Comment:
undefined/null values are now filtered out of the tasksResult array.

Now script execution will not stop at this point if an error occurs 🤞

Reply:
🤔